content/reference-rippled.md

```diff
@@ -7571,6 +7750,247 @@ In addition to the standard Offer fields, the following fields may be included i
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `channel_id` | String | The unique ID of the payment channel to use.
+| `secret` | String | The secret key to use to sign the claim. This must be the same key pair as the public key specified in the channel. |
+| `amount` | String | Cumulative amount of XRP, in drops, to authorize. If the destination has already received a lesser amount of XRP from this channel, the signature created by this method can be redeemed for the difference. |
+
+**Note:** You cannot use Ed25519 keys to sign claims with this method. This is a known bug.
+
+#### Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```
+{
+    "id": "channel_authorize_example_id1",
+    "status": "success"
+    "result": {
+        "signature": "304402204EF0AFB78AC23ED1C472E74F4299C0C21F1B21D07EFC0A3838A420F76D783A400220154FB11B6F54320666E4C36CA7F686C16A3A0456800BBC43746F34AF50290064",
+    }
+}
+```
+
+*JSON-RPC*
+
+```json
+200 OK
+
+{
+    "result": {
+        "signature": "304402204EF0AFB78AC23ED1C472E74F4299C0C21F1B21D07EFC0A3838A420F76D783A400220154FB11B6F54320666E4C36CA7F686C16A3A0456800BBC43746F34AF50290064",
+        "status": "success"
+    }
+}
+```
+
+*Commandline*
+
+```
+{
+    "result": {
+        "signature": "304402204EF0AFB78AC23ED1C472E74F4299C0C21F1B21D07EFC0A3838A420F76D783A400220154FB11B6F54320666E4C36CA7F686C16A3A0456800BBC43746F34AF50290064",
+        "status": "success"
+    }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format](#response-formatting), with a successful result containing the following fields:
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `signature` | String | The signature for this claim, as a hexadecimal value. To verify this signature or process the claim, the destination of the payment channel must send a [PaymentChannelClaim transaction][] with this signature, the exact Channel ID, XRP amount, and public key of the channel. |
+
+#### Possible Errors
+
+* Any of the [universal error types](#universal-errors).
+* `badSeed` - The value specified in the `secret` field was not a valid secret key.
+* `channelAmtMalformed` - The value specified in the `amount` field was not a valid XRP amount. See [Specifying Currency Amounts](#specifying-currency-amounts) for details.
+* `channelMalformed` - The value specified in the `channel_id` field of the reqeuest was not a valid Channel ID. The Channel ID should be a 256-bit (64-character) hexadecimal string.
+
+
+
+## channel_verify
+[[Source]<br>](https://github.com/ripple/rippled/blob/d4a56f223a3b80f64ff70b4e90ab6792806929ca/src/ripple/rpc/handlers/PayChanClaim.cpp#L89 "Source")
+
+_(Requires the [PayChan amendment](concept-amendments.html#paychan) to be enabled. [New in: rippled 0.33.0][])_
+
+The `channel_verify` method checks the validity of a signature that can be used to redeem a specific amount of XRP from a payment channel.
```

Comment:
I'd argue your phrasing makes it less clear. It's easy to understand your phrasing as meaning that you use channel verify to redeem XRP. That's a key misunderstanding of the API.

One uses `channel_verify` to check the validity of a signature, so that one knows one is _able_ to redeem a claim if one decides to do so later. (One might decide not to do so, because one redeems a subsequent claim that supersedes this claim.)